### ISSUE TITLE
Cannot find python2 interpreter.

### ISSUE THREAD

**Issue by dongpvv** [RedisGraph/RedisGraph#2412]
run make error

> ubuntu 20.04

> apt-get install build-essential cmake m4 automake peg libtool autoconf

> git clone --recurse-submodules -j8 https://github.com/RedisGraph/RedisGraph.git

> make

error output

>  make -C /opt/RedisGraph/deps/RediSearch STATIC=1 BINROOT=/opt/RedisGraph/bin/linux-x64-release
deps/readies/mk/main:45: *** Cannot find python2 interpreter.
Use can install one by running 'make setup'..  Stop.
make[1]: *** [Makefile:296: /opt/RedisGraph/bin/linux-x64-release/search-static/libredisearch.a] Error 2
make: *** [Makefile:45: all] Error 2

**Comment by gkorland** [comment]
```
 Use can install one by running 'make setup'.. Stop.
```
Please try to run `make setup` first

**Comment by dongpvv** [comment]
> 
I tried.
    
    root@dev:/opt/RedisGraph# make setup
    make: *** No rule to make target 'setup'.  Stop.

**Comment by dongpvv** [comment]
I tried.

    root@dev:/opt/RedisGraph# make setup
    make: *** No rule to make target 'setup'.  Stop.

**Comment by chayim** [comment]
Okay - this looks like a bug with the current dependency set. If you have neither **python2** nor **python3** you'll see the error above. It seems that readies defaults to looking for python3.

If you install python3 only - you don't receive this.  Any chance you were working with a stock Ubuntu docker / or envrionment without python 3? I just did a fresh clone, fresh docker.. and reproduced this way.

tl;dr If you apt install python3 you're fine.  I've pushed a PR for the docs in the meantime.

**Comment by vityafx** [comment]
I am using ArchLinux and have both python versions installed, however, I still do get the same error, having just fetched the source code from master.

**Comment by matthewdarwin** [comment]
I have the same problem.

**Comment by rafie** [comment]
The right way to build RedisGraph:
```
git clone --recurse-submodules https://github.com/RedisGraph/RedisGraph.git
cd RedisGraph
./sbin/setup
make
```
The docs here are wrong and need to be fixed ASAP: https://redis.io/docs/stack/graph/
